packages/web/components/swap-tool/index.tsx

```diff
@@ -50,9 +52,9 @@ import { PromoDrawer } from "./promo-drawer";
 import { SplitRoute } from "./split-route";
 
 export const SwapTool: FunctionComponent<{
-  // IMPORTANT: Pools should be memoized!!
-  pools: ObservableQueryPool[];
-
+  /* IMPORTANT: Pools should be memoized!! */
+  memoedPools: ObservableQueryPool[];
+  dataLoading?: boolean;
```

Comment:
Perhaps we should rename this to `isDataLoading` to follow the boolean convention in this component